purchase_request/views/purchase_request_view.xml

```diff
@@ -92,6 +92,9 @@
                                 <field name="analytic_account_id"
                                        groups="analytic.group_analytic_accounting"/>
                                 <field name="date_required"/>
+                                <field name="estimated_cost" widget="monetary"/>
+                                <field name="currency_id" invisible="1"/>
+                                <field name="company_id" groups="base.group_multi_company" widget="selection"/>
```

Comment:
@hveficent remove `widget="selection"` and use `option={no_edit_create: true}`

Reply:
Which improvement we've got using this?

Reply:
Yes, If you using `widget="selection"` only see `80` records but if you `option={no_edit_create: true}`
then search more record.

Reply:
I think 80 records are fine for company related field. I'll keep it.

Reply:
@hveficent If you have possible then Change it.